.gitignore

```diff
@@ -8,6 +8,8 @@ __pycache__
 .pytest_cache
 *.pyc
 
+.mypy_cache/
```

Comment:
Hi @art3xa 

Thanks for your contribution but I don't see this directory being generated in my local. Is there some special case when it is generated?